src/cadenza.js

```diff
@@ -1311,6 +1313,9 @@ function array(/** @type unknown */ value) {
  * @property {TYPE} type - The event type
  * @property {DETAIL} detail - Optional event details (depending on the event type)
  */
+/**
+ * @typedef {CadenzaEvent<'action', {context?: string}>} CadenzaActionEvent - When the user executed a POST message action.
```

Comment:
Proposal: "... which is defined on an external link in the Cadenza management center."